pressbooks/pressbooks
# Pb 2353 contributor picture cropping

Related issue: https://github.com/pressbooks/pressbooks/issues/2353

This change fixes the WP media cropping settings for contributors' picture feature.
When the image is taller than larger and it's more than 400px of larger, the cropper fails due a pixel falls out of the image.
Previously there is not a check for minimal size (400px), there was just a suggestion. Now the the checking is performed from the back end and front end sides. Now, images smaller then 400px are not allowed to upload. If a image smaller than 400px is selected from the media library, it is not inserted for the picture and a error is displayed to inform the user.

### Test
- Add or edit a contributor in a book (Book Info > Contributors)
- Add an image larger than 400px and taller than larger. The cropper should work.
- Try to upload a image smaller than 400px (width and/or height). A message should be displayed indicating the minimum size is 400px.
- Try to add an image smaller than 400px from the media library (instead uploading from the contributors section)
- Try to add that image in any contributor's profile picture. An error message must be displayed.
